### PR TITLE
LIVE-2854 Tron integration test working again

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Make tron test working again

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledger-live-common/src/families/tron/bridge.integration.test.ts [hunk 1]
```diff
@@ -129,12 +129,13 @@ const tron: CurrenciesData<Transaction> = {
...
           }),
-          expectedStatus: {
-            amount: new BigNumber("9906000"),
-            errors: {},
-            warnings: {},
-            totalSpent: new BigNumber("10006000"),
-            estimatedFees: new BigNumber("100000"),
+          expectedStatus: (account, transaction, status) => {
+            return {
+              amount: account.spendableBalance.minus(status.estimatedFees),
```
Comment (hedi-edelbloute) [inline libs/ledger-live-common/src/families/tron/bridge.integration.test.ts:134]: Not sure if it's okay to add that much logic in integration tests

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledger-live-common/src/families/tron/bridge.integration.test.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledger-live-common/src/families/tron/bridge.integration.test.ts:135]: I'm afraid that fees can move and make our test mark as false negative

Comment (hedi-edelbloute) [reply libs/ledger-live-common/src/families/tron/bridge.integration.test.ts:135]: Ok then :+1:

**Comment by henrily-ledger** [comment]
CI seems to be broken, PR on hold
I wasn't able to reproduce the api flakyness maybe I was wrong. 
Test seems to be alright: 
https://github.com/LedgerHQ/ledger-live/actions/runs/2780931534

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**